stock_by_warehouse/models/product_product.py

```diff
@@ -18,6 +20,10 @@ class ProductProduct(models.Model):
         store=False, readonly=True,
```

Comment:
Why a store=False on a non-computed field? What am I missing?

Reply:
Is is a computed field to assign only at memory level... but good catch... @moylop260 was this part of your refactor or a mistake?